force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -26,184 +26,164 @@
      */
     public static List<Boolean> checkStringContent(String input) {
-        // Initialize a list to store the results
-        
-        // Check if the string starts with 'Sales'
-        
-        // Check if the string ends with 'Force'
-        
-        // Check if the string contains 'Awesome'
-        
-        return null; // Replace null with the variable you used to store the result
+        List<Boolean> result = new List<Boolean>();
+
+        Boolean startsWith = input.startsWith('Sales');
+        result.add(startsWith);
+
+        Boolean endsWith = input.endsWith('Force');
+        result.add(endsWith);
+
+        Boolean contains = input.contains('Awesome');
+        result.add(contains);
+    
+        return result; 
     }
 
     /**
-     * Question 3
      * Take a sentence then remove any duplicate words, and return the sentence with only unique words.
-     * Example: removeDuplicateWords('Hello Hello Salesforce Salesforce') should return 'Hello Salesforce'.
+     * Example: removeDuplicateWords('Hello, Hello, Salesforce!') should return 'Hello, Salesforce!'.
      * @param sentence The input sentence.
      * @return A string consisting of only unique words from the input sentence.
      */
     public static String removeDuplicateWords(String sentence) {
-        // Split the sentence into words
-
-        // Create a set to store the unique words
-
-        // Add each word to the set
-
-        // Join the unique words back into a sentence
-
-        // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        List<String> newList = new List<String>();
+        newList = sentence.split(' ');
+        Set<String> newSet = new Set<String>(newList);
+        String joinedString = String.join(new List<String>(newSet), ' ');
+        return joinedString; 
     }
 
     /**
-     * Question 4
      * This method takes a string as a parameter, and calculates the number of vowels in the string.
      * For simplicity, we will consider only five vowels: 'a', 'e', 'i', 'o', 'u', and we will ignore case.
      * Example: countVowels("Hello World!") should return 3
      * @param s The string in which to count vowels.
      * @return The number of vowels in the string.
      */
     public static Integer countVowels(String str) {
-        // Initialize the result Integer
+        Integer result = 0;
 
-        /*
-        for () {
-            // get individual characters from the string - hint use substring or split
-            // check if the character is a vowel
-            // if it is a vowel, increment the result Integer
+        for (Integer i = 0; i < str.length(); i ++) {
+            String currentChar = str.substring(i, i + 1);
+            if (currentChar == 'a' || currentChar == 'e' || currentChar == 'i' || currentChar == 'o' || currentChar == 'u') {
```

Comment:
What happens if `currentChar` is an uppercase vowel? Will it be counted? If not, how can we fix it?

Instead of this long if statement, how might you take advantage of the String or List class method `contains()`?